src/GNDStk/node2type/test/node2type.test.cpp

```diff
@@ -45,9 +45,15 @@ SCENARIO("Testing GNDStk convert(node,type)") {
       // container with the contents of the "text" metadatum's value.
       foo.add("pcdata").add("text","12 34 56 78 90");
 
+      // Update, 2020-10-19. The relevant convert() functionality has been
+      // reformulated into the detail::convert_pcdata_text_t callable object,
+      // so we'll reformulate with that. Ultimately, due to the location of
+      // the new code, we should probably put this test material elsewhere.
+
       THEN("convert(foo,deque) extracts the content correctly") {
          std::deque<int> container;
-         convert(foo,container);
+         // convert(foo,container);
```

Comment:
No worries. The refactoring (really, more than just that) was for a good reason, but one that would require a somewhat long explanation here. Suffice it to say that there was something that would have done the right thing most of the time, by default, but been more awkward for users to circumvent in the instances where users might wish to customize something and wanted that particular aspect of it to work differently.

I replaced this "usually right, but awkward to work around when wrong" with, roughly speaking, "undefined by default, but easier to work around; and still defined in the constructs GNDStk provides when you don't customize." Or something like that. If a user writes something custom without providing a function that's needed, they'll get something like "error: no such function" instead of some multi-page convoluted nonsense that ultimately means that my default does the wrong thing in their case.

Months ago someone suggested that I use the keywords: `or`, `and`, etc. They *are* standard now. However, I've done some checking around, and for certain reasons (for example, creating possibly confusion with languages that use the words but with precedence being different), they're generally considered to be inferior to the original way of doing them, which are clear to any C++ expert. So, I made a decision to change them back.

Ya, I'll remove the commented code. I guess I thought it might be worthwhile for someone to see what the new lines were, effectively, doing. But really, it just doesn't matter.